tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -754,12 +754,12 @@ public function testToArray()
         $model->age = null;
         $model->password = 'password1';
         $model->setHidden(['password']);
-        $model->setRelation('names', new BaseCollection([
+        $model->setRelation('names', new Collection([
```

Comment:
I changed the relation from Collection to Eloquent Collection. A genuine Eloquent relation could only be an instance of a Model or an Eloquent Collection.